crepr/crepr.py

```
@@ -377,5 +377,23 @@ def remove(
 
 
+@app.command()
+def report_missing(
+    files: Annotated[list[pathlib.Path], file_arg],
+) -> None:
+    """Report classes without __repr__ methods."""
+    for file_path in files:
+        try:
+            module = get_module(file_path)
+            report_missing_classes(module, file_path)
+	except CreprError as e:
```

Ruff

Indentation contains tabs

(W191)